src/features/auth/api/appointment.ts

```diff
@@ -0,0 +1,78 @@
+import { api } from '@/core/api';
+
+interface AppointmentData {
+  description: string;
+  creationDate: string;
+  updateDate: string;
+  dueDate: string;
+  categoryID: number;
+  authorID: number;
+}
+
+interface UpdatedAppointmentData {
+  description: string;
+  creationDate: string;
+  updateDate: string;
+  dueDate: string;
+  categoryID: number;
+  authorID: number;
+}
+
+export async function createAppointment(appointmentData: AppointmentData) {
+  const response = await api.post('/appointments', {
+    method: 'POST',
```

Comment:
non è necessario specificare il metodo visto che già lo abbiamo definito in api.post